api/admin/routes.py

```diff
@@ -46,9 +46,10 @@ def decorated(*args, **kwargs):
 def requires_csrf_token(f):
     @wraps(f)
     def decorated(*args, **kwargs):
-        token = app.manager.admin_sign_in_controller.check_csrf_token()
-        if isinstance(token, ProblemDetail):
-            return token
+        if flask.request.method in ["POST", "PUT", "DELETE"]:
```

Comment:
cross-site request forgery is an attack where someone goes to a page that submits a form to another site, so it only applies to requests that might be modifying data. It doesn't let the attacker see the result of a request. The token is a way to ensure that the user submitted the form from your site.

This code wasn't here before because we didn't have any routes that allowed both GET and POST.